grails-app/controllers/org/broadinstitute/orsp/api/NewConsentGroupController.groovy

```diff
@@ -45,51 +46,34 @@ class NewConsentGroupController extends AuthenticatedController {
     }
 
     def save() {
-        Issue consent;
+        Issue consent
+        ConsentCollectionLink consentCollectionLink
         try{
             List<MultipartFile> files = request.multiFileMap.collect { it.value }.flatten()
             User user = getUser()
-            String dataProject = request.parameterMap["dataProject"].toString()
             JsonParser parser = new JsonParser()
-            JsonArray dataProjectJson = parser.parse(dataProject)
+            JsonArray dataProjectJson = parser.parse(request.parameterMap["dataProject"].toString())
+            JsonArray dataConsentCollectionJson = parser.parse(request.parameterMap["dataConsentCollection"].toString())
```

Comment:
nit: not necessarily for this PR, but I really wish we had a better way of doing this, or at least packaging it into a utility class so we're not doing so much work in the controller. This pattern of pulling out a json array, then coercing into a domain object is very brittle. There is no error handling here which means if something is not correct in the `request.parameterMap`, the whole thing will fail with just a flash error message. 

It might make more sense to:
* compose the various objects from the request params
* inspect each of them for validity
* return a more useful error in the case that anything is not correct
* perform the save operation on the validated objects.